kodadot/nft-gallery
# Silent History component

- replace Jest with Vitest
- allow only test case
- vitest types
- adjust renovate config (#2)
- typo
- add python version
- adjust loglevel to warn
- test python version 3.6
- set python version to 3.7
- python 3.8
- update keyring
- revert npmrc and delete runtime.txt
- :bug: isOpen property was set badly in History

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type


### What's new?


### Before submitting Pull Request, please make sure:


### Optional


### Had issue bounty label?


### Community participation


### Screenshot
